package.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "@conflux/react-ui",
-  "version": "0.0.1",
+  "name": "@conflux-ui/react",
```

Comment:
why change name from react-ui to react?

Reply:
Cause there's `@conflux-ui` as the org name, `@conflux-ui/react-ui` is a bit noisy.